thamos/lib.py

```diff
@@ -1010,8 +1011,16 @@ def get_status(api_client: ApiClient, analysis_id: str = None):
 
 
 @with_api_client
-def get_analysis_results(api_client: ApiClient, analysis_id: str):
+def get_analysis_results(
+    api_client: ApiClient, analysis_id: typing.Optional[str] = None
+):
     """Get the analysis result from a given id."""
+    with open("/home/fpokorny/git/thoth-station/adviser/adviser_document.json") as f:
```

Comment:
it should be an env variable for path?

Reply:
That was a file used to run experiments. Removed. Thanks for spotting it 👍🏻